File: 02_Programming_Fundamentals/02_Programming_Fundamentals_with_C#/02_Intro_And_Basic_Syntax_Exercise/05_Login.cs
using System;

public class Login
{
	public static void Main()
	{
		string username = Console.ReadLine();
		string password = Reverse(username);
		string newPassword = "";
		int counter = 0;
		
		while(newPassword != password) 
		{
			newPassword = Console.ReadLine(); 
			
			if (newPassword != password) 
			{
				counter++;
				if (counter == 4) 
				{
					break;
				}
				Console.WriteLine("Incorrect password. Try again.");
			}
		}
		
		if (counter >= 4) 
		{
			Console.WriteLine("User {0} blocked!", username);
		} 
		else {
			Console.WriteLine("User {0} logged in.", username);
		}
	
	}
	
	public static string Reverse( string s )
	{
		char[] charArray = s.ToCharArray();
		Array.Reverse( charArray );
		return new string( charArray );
	}
  
}

File: 02_Programming_Fundamentals/04_Programming_Fundamentals_PHP/02_Intro_And_Basic_Syntax_Exercise/05_Login.php
<?php 

$username = readline();
$password = strrev($username);
$counter = 0;
$loginPassword = "";

  while($loginPassword != $password) {
    $loginPassword = readline();

    if ($loginPassword != $password) {
      $counter++;

      if ($counter == 4) { break; }
      echo "Incorrect password. Try again." . PHP_EOL;
    }
  }

  if ($counter >= 4) {
    echo "User " . $username . " blocked!" . PHP_EOL;
  } 
  else {
    echo "User " . $username . " logged in." . PHP_EOL; 
  }

?>